CHANGELOG.md

```diff
@@ -0,0 +1,146 @@
+# Unreleased
+- 
+
+# v2.8.0
+
+### Initialization
+
+- [added] The [`initialize_app()`](/docs/reference/admin/node/admin#.initializeApp)
```

Comment:
We use relative URLs (starting with `/docs/`) in the Firebase devsite, but that won't work in Github. Should we fix that?

Reply:
Oh yeah, I thought I had fixed those everywhere by putting in the full path.. I did search and replace so not sure how I missed this one or others, so let me double check and address